Qube TechRadar.csv

```diff
@@ -5,3 +5,4 @@ AWS DevOps,Trial,Platforms,FALSE,"AWS DevOps provides services to simplify provi
 Microservices,Adopt,Techniques,FALSE,"Microservices arranges an application as a collection of loosely coupled services."
 Yocto,Adopt,Tools,FALSE,"The Yocto Project (YP) is an open source collaboration project that helps developers create custom Linux-based systems regardless of the hardware architecture."
 SQLite,Adopt,Tools,FALSE,"SQLite is a C-language library that implements a small, fast, self-contained, high-reliability, full-featured, SQL database engine."
+Python,Adopt,languages-and-frameworks,False,"Python is an interpreted, high-level, general-purpose, open source programming language"
```

Comment:
If possible, we could add some context in the description to say why we think Python is a good option for our use-cases.

Reply:
@DeepakThi Agreed. The description now only explains what the technology is about. Anyone can find that information in google. Explaining why we have decided to place a technology in a ring will be really helpful.
@grkkarthick90 Can you please add those details.